From f74266c64cf197b9613fe6ee85cefa2f3841594f Mon Sep 17 00:00:00 2001
From: Enguerrand de Ribaucourt <enguerrand.de-ribaucourt@savoirfairelinux.com>
Date: Tue, 10 Dec 2024 14:03:12 +0100
Subject: [PATCH] ping: remove root suid/sgid

Upstream-Status: Inappropriate [Seapath specific]
Signed-off-by: Enguerrand de Ribaucourt <enguerrand.de-ribaucourt@savoirfairelinux.com>
---
 ping/Makefile.am | 2 --
 1 file changed, 2 deletions(-)

diff --git a/ping/Makefile.am b/ping/Makefile.am
index edf83fc..fa0f3cf 100644
--- a/ping/Makefile.am
+++ b/ping/Makefile.am
@@ -41,8 +41,6 @@ ping6_SOURCES = ping6.c ping_common.c ping_common.h ping6.h
 ping6_LDADD = $(top_builddir)/libicmp/libicmp.a $(LDADD) $(LIBSOCKET) \
   $(HOSTENT_LIB) $(LIBINTL) $(GETADDRINFO_LIB)
 
-SUIDMODE = -o root -m 4755
-
 install-ping-hook:
 	-@for x in $(bin_PROGRAMS); do \
 	$(INSTALL_PROGRAM) $(SUIDMODE) $(AM_INSTALL_PROGRAM_FLAGS) $$x $(DESTDIR)$(bindir)/`echo $$x|sed '$(transform)'` ; OUTCOME=$$?; \
